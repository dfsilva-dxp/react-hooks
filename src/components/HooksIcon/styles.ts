import styled from "styled-components";

export const HooksIconWrapper = styled.div`
  width: 2rem;
  min-width: 2rem;

  svg {
    width: 100%;
    block-size: fit-content;
  }
`;
