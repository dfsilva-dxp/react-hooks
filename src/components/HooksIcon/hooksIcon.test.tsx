import { render, screen } from "@testing-library/react";

import HooksIcon from ".";

describe("<Card/>", () => {
  it("should render currectly", () => {
    const { container } = render(<HooksIcon />);

    expect(
      screen.getByLabelText("React Hooks Icon Learning")
    ).toBeInTheDocument();

    expect(container.parentElement).toMatchSnapshot();
  });
});
