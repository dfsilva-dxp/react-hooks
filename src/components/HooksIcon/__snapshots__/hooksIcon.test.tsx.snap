// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> should render currectly 1`] = `
.c0 {
  width: 2rem;
  min-width: 2rem;
}

.c0 svg {
  width: 100%;
  -webkit-block-size: fit-content;
  -ms-flex-block-size: fit-content;
  block-size: fit-content;
}

<body>
  <div>
    <div
      class="c0"
    >
      <svg
        aria-label="React Hooks Icon Learning"
        fill="none"
        height="44"
        preserveAspectRatio="none"
        role="img"
        viewBox="0 0 50 44"
        width="50"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M46.1 18.7C36.3 22.3 21 27.9 11.2 31.5L11.5 29.7C6.5 27.6 2 25.8 2.8 20.6V20.3C14.1 16.2 27 11.4 38.4 7.29999C38.4 7.69999 38.5 8.09999 38.5 8.39999C38.6 10.3 38.4 12.1 38 14C41 15 44.1 16.3 46.1 18.7V18.7Z"
          fill="#00875F"
        />
        <path
          d="M38.4 7.79999C27.1 11.9 14 16.7 2.7 20.9C2.7 20.8 2.7 20.7 2.8 20.6V20.3H2.9C4.5 17 8.1 15.5 11.5 14.2C10.4 9.79999 10.5 2.19999 15.9 1.99999C19.3 1.89999 22.3 4.09999 24.8 6.29999C29.5 2.69999 38.2 -1.40001 38.4 7.79999V7.79999Z"
          fill="#00B37E"
        />
        <path
          d="M3.6 24.9C2.7 23.6 2.3 22.1 2.8 20.6C5 15.6 11.2 14.1 16.2 13M3.6 24.9C5.8 27.1 8.6 28.9 12.7 30.1M38.9 29.6C39.8 29.3 40.7 28.9 41.6 28.5C43.6 27.5 45.6 26.3 46.7 24.4C46.8 24.3 46.8 24.1 46.9 24C49.2 18.6 42.2 15.3 38 14C37.2 13.8 36.5 13.6 35.7 13.4"
          stroke="#202024"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-miterlimit="10"
        />
        <path
          d="M13.2 24.1C11.5 28.9 9.3 36.5 12.8 40.6C18.6 45.2 27 36.5 30.2 31.7M24.9 6.4C39.2 -4.6 40.2 8 36.8 18.9M38 39.5C40.3 35.8 38.5 24.2 35.9 21.9M22.1 34.6C28.3 41 34.4 44.8 38 39.5"
          stroke="#202024"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-miterlimit="10"
        />
        <path
          d="M12.3 17.4C8.1 6.40003 13.5 -4.19997 24.8 6.30003C26.4 7.70003 27.9 9.30003 29.3 11.1"
          stroke="#202024"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-miterlimit="10"
        />
        <path
          d="M16.7 2.09998C19.1 2.09998 22 3.39998 24.8 6.29998H25C28.3 3.59998 31.3 2.29998 33.6 2.29998C37.6 2.29998 39.7 6.29998 38 14.1C41.8 15.4 47.1 17.4 47.4 22C47.2 26.7 42 27.9 38.9 30C39.8 38.1 37.2 41.9 33.4 41.9C31 41.9 28 40.4 25 37.6C21.9 40.6 19 42 16.5 42C12.2 42 9.6 37.5 11.5 29.8C-0.7 26.1 0.5 16.8 12 14.3C9 6.89997 12 2.09998 16.7 2.09998M16.7 0.0999756C14 0.0999756 11.7 1.29998 10.3 3.39998C8.7 5.79998 8.4 9.09998 9.4 12.8C4.2 14.5 1 17.8 0.8 21.6C0.7 25.5 3.8 29 9.2 31C8.5 35.3 9 38.8 10.9 41.1C12.3 42.8 14.3 43.8 16.6 43.8C19.3 43.8 22.2 42.5 25.1 40.1C28 42.5 30.8 43.7 33.4 43.7C35.4 43.7 37.2 42.9 38.5 41.5C40.5 39.3 41.3 35.7 41 30.9C41.4 30.6 41.9 30.4 42.4 30.1C45.4 28.5 49.2 26.5 49.4 21.9V21.7C49 16.4 43.9 14 40.3 12.6C41 8.09998 40.5 4.59998 38.8 2.49998C37.6 0.999976 35.8 0.199976 33.7 0.199976C31.2 0.199976 28.2 1.39998 25.1 3.69998C22.4 1.39998 19.5 0.0999756 16.7 0.0999756V0.0999756Z"
          fill="#202024"
        />
        <path
          d="M19.2 24.9C19.1 27.7 17.5 26.9 15.9 26.6V29.4C24.9 32.6 21.9 19 22.4 14.4H19.2V24.9ZM33.8 23.4C33.1 21.5 30 20.8 28.4 19.8C25.5 16.8 31.1 16.6 33.1 18L34.1 15.3C29.7 12.5 21.6 15.4 25.3 21.1C25.8 22.2 30 23.6 30.6 24.3C32.8 27.9 26.2 27 24.3 25.6V28.7C28.1 31.1 36.2 29.2 33.8 23.4Z"
          fill="#323238"
        />
      </svg>
    </div>
  </div>
</body>
`;
